classroom/snippets/createCourse.gs

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
You can put all of these snippets into one file, perhaps called `courseSnippets.gs`.